theme/beevibrant/static/css/main.css

```diff
@@ -56,12 +56,7 @@ h3 { font-size: 24px;}
 h4 { font-size: 18px;}
 h5 { font-size: 16px;}
 h6 { font-size: 14px;}
-h1 a,
```

Comment:
Please don't change the CSS in this file. Let's keep the scope to the original Google search thing.
We can always improve it on other PR (I really think that it would need it).

Reply:
this comes from the introducing headers in `content/pages/index.md`, without this change the links will not be visible as links, so we'll fix the search engines, but human beings visiting the landing page would not be able to see that "Schedule", "Tickets" and "the next edition" are links

Reply:
Would you mind including this as inline style? Is that a good compromise between the two?

Reply:
I'm not sure if this is possible, because the style is applied by pelican after generating these H-tags from markdown.

The intent of the original CSS was to make clickable H-tags keeping the original color. AFAIK, we don't have any clickable html H tags except for Keynote speakers on the main page, where you see their names in the news article. Before this change their names are grey and when you hover over them they become blue. After that change they'll always will be blue. This is the only side-effect I'm aware of.